### ISSUE TITLE
Remove deprecated getPythonContainers

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#7920]
|[<img alt="hawkowl's avatar" src="https://avatars.githubusercontent.com/u/3307100?s=50" width="50" height="50">](https://github.com/hawkowl)| @hawkowl reported|
|-|-|
|Trac ID|trac#7920|
|Type|task|
|Created|2015-05-29 15:29:47Z|
|Branch|https://github.com/twisted/twisted/tree/remove-getpythoncontainers-7920|

It's been deprecated and doesn't really work, according to the comment. Let's remove it.

<details><summary>Searchable metadata</summary>

```
trac-id__7920 7920
type__task task
reporter__hawkowl hawkowl
priority__normal normal
milestone__ 
branch__branches_remove_getpythoncontainers_7920 branches/remove-getpythoncontainers-7920
branch_author__hawkowl hawkowl
status__closed closed
resolution__fixed fixed
component__trial trial
keywords__ 
time__1432913387343835 1432913387343835
changetime__1433407569126886 1433407569126886
version__None None
owner__hawkowl hawkowl
cc__jml
```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="hawkowl's avatar" src="https://avatars.githubusercontent.com/u/3307100?s=50" width="50" height="50">](https://github.com/hawkowl)<a name="note_2"></a>|@hawkowl commented|
|-|-|

(In [44972]) Branching to remove-getpythoncontainers-7920.

**Comment by twisted-trac** [comment]
|[<img alt="hawkowl's avatar" src="https://avatars.githubusercontent.com/u/3307100?s=50" width="50" height="50">](https://github.com/hawkowl)<a name="note_3"></a>|@hawkowl commented|
|-|-|

Changes done, builders spun. Hey, that rhymed.

**Comment by twisted-trac** [comment]
|[<img alt="adiroiban's avatar" src="https://avatars.githubusercontent.com/u/204609?s=50" width="50" height="50">](https://github.com/adiroiban)<a name="note_4"></a>|@adiroiban set owner to @hawkowl|
|-|-|

I think that you have accidentally removed a line... as the test fail

```
-@defer.inlineCallbacks		
 def _runSequentially(callables, stopOnFirstError=False):
```

please review the diff and check buildbot results.

thanks!

**Comment by twisted-trac** [comment]
|[<img alt="hawkowl's avatar" src="https://avatars.githubusercontent.com/u/3307100?s=50" width="50" height="50">](https://github.com/hawkowl)<a name="note_5"></a>|@hawkowl removed owner|
|-|-|

Whoops, that was by accident.

Fixed, builders spun again.

**Comment by twisted-trac** [comment]
|[<img alt="hynek's avatar" src="https://avatars.githubusercontent.com/u/41240?s=50" width="50" height="50">](https://github.com/hynek)<a name="note_6"></a>|@hynek set owner to @hawkowl|
|-|-|

Buildbots are green and deprecated function is gone.

Please merge.

**Comment by twisted-trac** [comment]
|[<img alt="hawkowl's avatar" src="https://avatars.githubusercontent.com/u/3307100?s=50" width="50" height="50">](https://github.com/hawkowl)<a name="note_7"></a>|@hawkowl set status to `closed`|
|-|-|

(In [45005]) Merge remove-getpythoncontainers-7920: Remove the deprecated getpythoncontainers

Author: hawkowl
Reviewers: adiroiban, hynek
Fixes: [#7920](https://github.com/twisted/twisted/issues/7920)
